metricflow/test/plan_conversion/test_dataflow_to_sql_plan.py

```diff
@@ -1035,3 +1036,56 @@ def test_dimensions_requiring_join(
         sql_client=sql_client,
         node=dataflow_plan.sink_output_nodes[0].parent_node,
     )
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.sql_engine_snapshot
+def test_nested_offsets(  # noqa: D
```

Comment:
Can we put these into the test_derived_metric_rendering.py module instead? I'd like to keep this one more strictly limited to simple component conversion rather than query rendering.

Reply:
Yes my bad!